mantidproject/mantid
# Apply changes from clang-tidy readability-container-size-empty.

Description of work.

The applies changes from the clang-tidy check [readability-container-size-empty](http://clang.llvm.org/extra/clang-tidy/checks/readability-container-size-empty.html). It looks for cases where `.size()` can be replaced with `.empty()`.

**To test:**



Review changes and check that the builds still pass.

This is a small change with no issue number.

Does not need to be in the release notes.

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
